always_encrypted_test.go

```diff
@@ -0,0 +1,104 @@
+package mssql
+
+import (
+	"fmt"
+	"testing"
+	"github.com/stretchr/testify/assert"
+	"time"
+)
+
+type testAEStruct struct {
+	Id int
+	SSN string
+	Date time.Time
+	Float *float64
+	Money *float64
+}
+
+func TestAlwaysEncrypted(t *testing.T) {
+	conn := open(t)
+	defer conn.Close()
+	rows, err := conn.Query("SELECT id, ssn, secure_date, secure_float, secure_money FROM [dbo].[cid]")
```

Comment:
Move defer rows.Close after error check

Reply:
Where is `dbo.cid` coming from? If this is a unit test, it should be created first as part of the test.